src/Cloud/weglide/UploadFlight.hpp

```diff
@@ -29,17 +29,17 @@ Copyright_License {
 
 #include <cstdint>
 
-struct WeGlideSettings;
 class Path;
 class CurlGlobal;
 class ProgressListener;
 
 namespace WeGlide {
+struct Pilot;
 
 Co::Task<boost::json::value>
-UploadFlight(CurlGlobal &curl, const WeGlideSettings &settings,
+UploadFlight(CurlGlobal& curl,  const char *url, const struct Pilot &pilot,
              uint_least32_t glider_type,
              Path igc_path,
-             ProgressListener &progress);
+             ProgressListener& progress);
```

Comment:
Another `&` moved to the wrong position. Please do not change the coding style of existing code.